### PR TITLE
add axios

### PR DESCRIPTION
**Pull request by leonlafa**

### CODE REVIEW

File: client/src/components/TaskList.js [hunk 1]
```diff
@@ -1,44 +1,31 @@
 import React, { Component } from 'react';
+const { default: taskService } = require('../services/task.service.js')
```
Comment (jmsv) [inline client/src/components/TaskList.js:2]: could this be an ES6 import? e.g.
`import taskService from '../services/task.service.js'`

File: client/src/services/api.service.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+/**
+ * Instantiate axios instance
+ */
+
+const api = axios.create({
+    baseURL: '/api'
```
Comment (jmsv) [inline client/src/services/api.service.js:8]: please could you format the new files with 2-space indentation?

**Review by jmsv — CHANGES_REQUESTED** [review]

**Comment by jmsv** [comment]
could you also change the PR's target branch to `with-axios`? thanks 😀

File: client/src/components/TaskList.js (hunk 1 above)
Comment (leonlafa) [reply client/src/components/TaskList.js]: I sure can 😁

File: client/src/services/api.service.js (hunk 2 above)
Comment (leonlafa) [reply client/src/services/api.service.js]: Yeah of course. I think my VSCode defaulted to 4 for some reason.

**Review by leonlafa — COMMENTED** [review]
> could you also change the PR's target branch to `with-axios`? thanks 😀

Done 😁

**Review by jmsv — APPROVED** [review]

**Comment by jmsv** [comment]
lgtm! thanks 🚀

**Pull request merged**